simapp/upgrades.go

```diff
@@ -50,3 +56,14 @@ func (app SimApp) RegisterUpgradeHandlers() {
 		app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
 	}
 }
+
+func (app SimApp) syncAccountNumber(ctx context.Context) error {
```

Comment:
godoc

Reply:
how come this is in upgrade instead of auth? that should be documented as well

Reply:
moved to auth